pnpm-workspace.yaml

```diff
@@ -1,2 +1,7 @@
 packages:
-  - 'packages/*'
+  - packages/*
+onlyBuiltDependencies:
+  - edgedriver
+  - esbuild
+  - geckodriver
+  - puppeteer
```

Comment:
not sure what those are... pnpm asked me something and i just said yes.
let me know if u want them removed.